packages/design-system/src/components/contextMenu/components/button.js

```diff
@@ -32,10 +32,19 @@ const StyledButton = styled(BaseButton)`
   ${menuItemStyles};
 
   width: 100%;
-  padding: ${({ $isIconMenu }) => ($isIconMenu ? 0 : '2px 16px')};
-  border-radius: ${({ $isIconMenu }) => ($isIconMenu ? 4 : 0)}px;
   background-color: transparent;
 
+  ${({ $isHorizontal, $isIconMenu, theme }) => {
+    const hasHeight = $isHorizontal; // In a horizontal menu, all buttons are 32px high
+    const hasNoPadding = $isHorizontal || $isIconMenu;
+    const hasBorderRadius = $isHorizontal || $isIconMenu;
+    return {
+      height: hasHeight && '32px',
+      padding: hasNoPadding && 0,
```

Comment:
Should it be `padding: hasNoPadding ? 0 : '2px 16px'`? Or is that the default?

Reply:
The other padding is default, so no need to redefine it. This is just about removing padding or not.